scripts/koji/koji_build.py

```diff
@@ -1,204 +1,9 @@
 #!/usr/bin/env python3
-import argparse
-import logging
-import os
-import re
-import subprocess
-from contextlib import contextmanager
-from datetime import datetime, timedelta
-from pathlib import Path
 
-try:
-    from specfile import Specfile
-except ImportError:
-    print("error: specfile module can't be imported. Please install it with 'pip install --user specfile'.")
-    exit(1)
+import sys
 
-TIME_FORMAT = '%Y-%m-%d-%H-%M-%S'
+from koji_utils.koji_build import main
```

Comment:
Could have a comment (and possibly a note printed) telling it is a now deprecated compatibility script?

Alternatively, just using symlinks would be simpler and likely more clear on which script version gets launched (the one in the same repo or the one previously installed).  It would also allow to relocate all the scripts into the new package first, and then add the entrypoints.

Reply:
> Could have a comment (and possibly a note printed) telling it is a now deprecated compatibility script?

like the one 2 lines below?

> Alternatively, just using symlinks would be simpler and likely more clear on which script version gets launched (the one in the same repo or the one previously installed). It would also allow to relocate all the scripts into the new package first, and then add the entrypoints.

I've also considered that, but printing a warning is simpler this way